performance/requirements.txt

```
@@ -0,0 +1,11 @@
+altair
+jupyter_server
+jupyterhub
+# usually get proxy via `-e .`, but this avoids triggering reinstall every time
+jupyterhub_traefik_proxy
+numpy
```

```suggestion
matplotlib
numpy
```
matplotlib is required by the notebok